draft-ietf-deleg.md

```diff
@@ -100,23 +100,23 @@ Terminology regarding the Domain Name System comes from {{?BCP219}}, with additi
 
 The DELEG record (whose RRtype is TBD) Rdata contains a list of key-value pairs called "delegation information".
 The delegation information has wire and display formats that are based on the rules in Appendix A of {{?RFC9460}}.
-A DELEG record is authoritative for the named zone, and creates a delegation and thus lives in the parent of the named zone.
+A DELEG record is authoritative for the zone in its owner name, and creates a delegation and thus lives in the parent of that zone.
 
 The DELEGI record uses the same format as the DELEG record, but rather than defining a delegation,
 a DELEGI is used as the target of the "include" mechanism.
-Also unlike DELEG records, DELEGI records can live in any part of the DNS namespace unrelated to the delegating domain.
+Also unlike DELEG records, DELEGI records can live in any part of the DNS namespace.
 
 When a resolver encounters an "include-name" key-value pair in a DELEG record, it queries for a DELEGI record.
 Because a DELEGI record can itself contain an "include-name" key-value pair,
 a resolver must be prepared to follow the "include-name" chain by making additional DELEGI queries.
 
-Some delegation information key-value pairs are actions that a DELEG-aware resolver takes when it gets a DELEG or DELEGI record.
-The actions defined in this document are described briefly here, and more fully described in {{actions}}.
+Some delegation information key-value pairs are information about nameservers that a DELEG-aware resolver uses when it gets a DELEG or DELEGI record.
+The keys defined in this document are described briefly here, and more fully described in {{nameserver-info}}.
 
-* server-ip4: a set of IPv4 addresses for nameservers of the given zone
-* server-ip6: a set of IPv6 addresses for nameservers of the given zone
-* server-name: the domain name of a nameserver of the given zone
-* include-name: the domain name of a zone that has more information about the nameservers of the given zone
+* server-ip4: a set of IPv4 addresses for nameservers
+* server-ip6: a set of IPv6 addresses for nameservers
```

Comment:
In version 01 people complained about missing "v" and perceived inconsistency with SVCB's 'ipv?hint' notation, and this was repeated in review of 02 by Miek.

Is omission of 'v' intentional here?